pkg/commands/options/build.go

```diff
@@ -51,8 +51,10 @@ type BuildOptions struct {
 	ConcurrentBuilds     int
 	DisableOptimizations bool
 	SBOM                 string
-	Platforms            []string
-	Labels               []string
+	// SBOMDir is the directory to write the SBOM to.
```

Comment:
```suggestion
```

I don't think the comment adds much, and it's weird to have a comment on this and nothing else.